src/app/components/client/Chart.tsx

```diff
@@ -89,11 +90,7 @@ export const DoughnutChart = (props: Props) => {
         fill="none"
         strokeWidth={ringWidth}
         strokeDasharray={`${circumference} ${circumference}`}
-        style={
-          {
-            "--sliceLength": circumference * (1 - percent) + sliceBorderWidth,
-          } as CSSProperties
-        }
+        strokeDashoffset={`${sliceLength}`}
```

Comment:
We can pass the prop directly:
```suggestion
        strokeDashoffset={sliceLength}
```